README.MD

```diff
@@ -183,10 +183,16 @@ sudo tar -xzf bbb_nfs.tar.gz
 
 ## 2.4 Configuring host PC
 
-Service wise there is different ways to do everything but the end goal is
-to have static IP address on the secondary network interface (_192.168.30.1_)
-and to have DHCP, TFTP and NFS server running on that network. It is very important that you are careful especially with the DHCP server so that you don't accidentally configure it to the wrong interface (you should never configure the DHCP server to the interface from which your PC Host gets its internet, unless you know exactly what you are doing. This guide has
-been written using Ubuntu so on different distro you might have to do things
+In this section, we set up the necessary services to enable the devices to communicate with and
+access files on the host PC. The phases here correspond to [BBB booting overview](#22-bbb-booting-overview) steps 2-5
+but are solely to enable the required services on host PC.
+In terms of services there are different ways to do everything but the end goal is
+to have a static IP address on the secondary network interface (_192.168.30.1_)
+and to have DHCP, TFTP and NFS server running on that network.
+
+
+It is very important that you are careful especially with the DHCP server so that you don't accidentally configure it to the wrong interface (you should never configure the DHCP server to the interface from which your PC Host gets its internet, unless you know exactly what you are doing. This guide has
```

Comment:
Could you put the other ")" after "...unless you know exactly what you are doing"